simplq/src/components/pages/Admin/AddMember.jsx

```diff
@@ -38,13 +33,15 @@ export default (props) => {
         <ExpandButton />
```

Comment:
Can you also switch the arrow up and down, it would be nice.

Reply:
sure. will do that